package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "snyk-protect": "snyk protect",
-    "test": "snyk test && node make test"
+    "test": "node make test"
```

Comment:
Yeah, I had the same issue. I just ran `node make test` locally, but I meant to come back to it. Forgot.